ts/app.ts

```diff
@@ -79,6 +87,28 @@ module MRoot {
             $scope.getAccountName = () => {
                 return $cookies.accountName;
             };
+            
+            $scope.changePassword = (win : kendo.ui.Window)=>{
+                $scope.modelChangePassword = new ChangePasswordModel();
+                win.center();
+                win.open();
+            }
+
+            $scope.funcChangePasswordDone = (win : kendo.ui.Window)=>{
+                var msg = new ApiHeader.APIUpdateAccountMsg();
+                msg.uuid = '36c27e8ff05c4780bf6d2fa65700f22e';

```

Comment:
We can save the accountUuid in browser localStorage. So you can easily get it. Maybe cookies is also okay.

We need to save User or Account information when user login. Then we can know which API should be called. 

But currently we only allow Account login in zstack-dashboard. So maybe we can just consider the Account.